packages/webapp/pages/onboarding.tsx

```diff
@@ -438,109 +438,108 @@ export function OnboardPage(): ReactElement {
   }
 
   return (
-    <div
-      className={classNames(
-        'z-3 flex h-full max-h-dvh min-h-dvh w-full flex-1 flex-col items-center overflow-x-hidden',
-        layout.hasCta && 'fixed',
-      )}
-    >
-      {showOnboardingPage && (
-        <img
-          alt="Onboarding background"
-          className="pointer-events-none absolute inset-0 -z-1 h-full w-full object-cover tablet:object-center"
-          fetchPriority="high"
-          loading="eager"
-          role="presentation"
-          src={onboardingVisual.fullBackground.mobile}
-          srcSet={`${onboardingVisual.fullBackground.mobile} 450w, ${onboardingVisual.fullBackground.desktop} 1024w`}
-          sizes="(max-width: 655px) 450px, 1024px"
-        />
-      )}
-      <Toast autoDismissNotifications={autoDismissNotifications} />
-      {showGenerigLoader && <GenericLoader />}
-      <OnboardingHeader
-        showOnboardingPage={showOnboardingPage}
-        setAuth={setAuth}
-        customActionName={customActionName}
-        onClick={onClickCreateFeed}
-        activeScreen={activeScreen}
-        showPlusIcon={isPlusCheckout && activeScreen === OnboardingStep.Plus}
-      />
+    <PaymentContextProvider>
```

Comment:
Wrapper everything with `PaymentContextProvider` since we need to change the header based on free trial field

Reply:
I was hesitant at first, but yeah maybe, since Plus is now for everyone, we can always request it. Though would love to hear others' opinions.